generators/beagle/src/main/scala/TestHarness.scala

```diff
@@ -75,15 +75,18 @@ class BeagleTestHarnessInner(implicit p: Parameters) extends LazyModule
 
     dut.reset := reset
     dut.boot := true.B
-    dut.alt_clks.foreach { _ := clock }
-    dut.alt_clk_sel := 0.U
+    dut.single_clks.foreach { _ := clock }
+    dut.diff_clks.foreach { _ := DontCare }
```

Comment:
I think with the behavioral model you could set `diff_clks.n := clock` and `diff_clks.p := !clock`